infra/modules/database/dms-task.tf

```diff
@@ -0,0 +1,301 @@
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dms_replication_task
+resource "aws_dms_replication_task" "task" {
+  replication_instance_arn = aws_dms_replication_instance.instance.replication_instance_arn
+  source_endpoint_arn      = aws_dms_endpoint.source_endpoint.endpoint_arn
+  target_endpoint_arn      = aws_dms_endpoint.target_endpoint.endpoint_arn
+  migration_type           = "full-load"
```

Comment:
I think we want `full-load-and-cdc` so that it loads fully once and then is continuious afterwards. Probably requires a few more settings as well

We also want to be certain that `TargetTablePrepMode` in the replication settings is `DO_NOTHING` (the others either drop the tables, or truncate the data) - we already have the tables so don't need to prep them.

Reply:
Updated - https://github.com/HHS/simpler-grants-gov/pull/1138/commits/bf20e6e49668b6ceec091d4a459898fb6d03a2dc, https://github.com/HHS/simpler-grants-gov/pull/1138/commits/54c1c58d379599af9f3602466b6ec5aa2deab550